static/app/components/updatedEmptyState.tsx

```diff
@@ -143,32 +41,83 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
     }
   );
 
+  const {isLoading: isLoadingRegistry, data: registryData} =
+    useSourcePackageRegistries(organization);
+
+  const currentPlatformKey = project?.platform ?? 'other';
+  const currentPlatform = platforms.find(
+    p => p.id === currentPlatformKey
+  ) as PlatformIntegration;
+
+  const platformPath = getPlatformPath(currentPlatform);
+
+  useEffect(() => {
+    async function getGettingStartedDoc() {
```

Comment:
I'm seeing this save useEffect in a few places, would it be possible to make this a common util?

Reply:
will merge a separate PR to do this first, then update this one !